packages/runtime/runtime-utils/src/summarizerNodeContract.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { ReadAndParseBlob, ISummarizerNodeRootContract } from "./summarizerNodeUtils";
```

Comment:
something to consider. move all these files into a folder, and rename this to index.ts. consumers then import the folder. that pattern of pseudo submodule in used around the code base.

Reply:
the main index.ts would then export the folder

Reply:
+1 to this. I think summarizerNode has a lot of files to warrant a folder of its own.
You might also consider a types.ts file which includes all interfaces to be exported and export that file in index.ts.

Reply:
it might be tricky to bundle this with the refactor. might was to push a small change that puts the exsitng files in the folder without modification and adds the index.ts. then apply these change on top

Reply:
I agree to having a folder (internal module), and that's how I originally was doing it.

We have an eslint rule preventing it: `eslint(import/no-internal-modules)`. I can add a line to ignore the eslint rule, but then it raises the question, why do we have the eslint rule at all?

Maybe the rule is meant to be for external modules? I see some places we are ignoring the rule elsewhere in the code.

Reply:
I went ahead and made the change, since I also prefer it.